app/controllers/api_docs/vendor_api_docs/pages_controller.rb

```diff
@@ -27,12 +27,25 @@ def when_emails_are_sent; end
 
     private
 
-      def render_content_page(page_name)
-        @converted_markdown = GovukMarkdown.render(File.read("app/views/api_docs/vendor_api_docs/pages/#{page_name}.md")).html_safe
+      def render_content_page(page_name, locals: {})
+        raw_content = File.read("app/views/api_docs/vendor_api_docs/pages/#{page_name}.md")
+        content_with_erb_tags_replaced = ApplicationController.renderer.render(
+          inline: raw_content,
+          locals: locals,
+        )
+        @converted_markdown = GovukMarkdown.render(content_with_erb_tags_replaced).html_safe
         @page_name = page_name
 
         render 'rendered_markdown_template'
       end
+
+      def current_api_version
+        AllowedCrossNamespaceUsage::VendorAPIInfo.production_version.to_f
+      end
+
+      def next_api_version
+        (current_api_version + 0.1).round(1)
```

Comment:
might be worth moving this to version helpers and testing 🤔

Reply:
Do we need `round(1)` here?

Reply:
Also what about if the next version is `v2`? I think it's better to specify the current version and maybe the lowest available version, as those are the actual relevant parameters

Reply:
> Do we need round(1) here? The only rounding issue I can think of is if current_api_version is specified at 2 decimal places. eg. 1.11 or is there a different case?

@steventux this surprised me as well, however this is what I got when not using the `round(1)`: 
```
[1] pry(main)> VendorAPI.production_version.to_f + 0.1
=> 1.2000000000000002
```
> Also what about if the next version is v2? If we wanted to be really granular we could say the next minor api version
@SamJamCul we aren't looking to prep for v2 yet as when we do there will be a major rework of both the documentation and code and we haven't thought through the structure of that yet. However good point re minor version will amend